Assets/Editor/DecentralandEditor.cs

```diff
@@ -147,14 +147,38 @@ public class DecentralandEditor : EditorWindow {
 
 		while (! www.isDone) {} // busy wait
 
-		if (string.IsNullOrEmpty(www.error)) {
-			RPCResponse response = JsonUtility.FromJson<RPCResponse>(www.text);
-			Debug.Log("Successfully published tile!");
-			published = true;
-		} else {
-			Debug.Log("Error publishing tile! " + www.error);
-			publishError = true;
-		}
+    // Web transaction error
+    if(!string.IsNullOrEmpty(www.error)) {
```

Comment:
Agreed, I'd better be consistent. If a convention is established I'll follow it.